Home_work25/Home_work25.cpp

```diff
@@ -0,0 +1,69 @@
+﻿// Home_work25.cpp : Этот файл содержит функцию "main". Здесь начинается и заканчивается выполнение программы.
+//
+
+#include <iostream>
+#include <vector>
+#include <thread>
+#include <mutex>
+#include <chrono>
+
+void arrSum(std::vector<int>& vec, int thread) {
+    int sum = 0;
+    int step = vec.size() / thread;
+    int k = 0;
+    std::vector<std::thread> threads;
+    
+    std::mutex m;
```

Comment:
mutex is not needed for this task and is highly inefficient. 
I'm impressed you have some boost after all)
Think of solution without any synchronization